inception-scheduling/src/main/java/de/tudarmstadt/ukp/inception/scheduling/config/SchedulingProperties.java

```diff
@@ -44,6 +44,6 @@ public int getQueueSize()
 
     public void setQueueSize(int aQueueSize)
     {
-        queueSize = queueSize;
+        queueSize = aQueueSize;
```

Comment:
Hm, this might be actually a genuine bug that you fixed! However, it is not related to your refactoring...